File: Hivemind/Contracts/GangerBattleStats.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Hivemind.Contracts
{
    public class GangerBattleStats
    {
        public string GangerId { get; set; }
        public int Kills { get; set; }
        public int Objectives { get; set; }
        public bool Down { get; set; }
        public bool OutOfAction { get; set; }
    }
}

File: Hivemind.Ng/src/autogenerated/entities/GangerBattleStats.ts

export class GangerBattleStats {
    public gangerId: string;
    public kills: number;
    public objectives: number;
    public down: boolean;
    public outOfAction: boolean;

    public constructor(partial: Partial<GangerBattleStats>) {
        if (partial.gangerId) {
            this.gangerId = partial.gangerId;
        }
        if (partial.kills) {
            this.kills = partial.kills;
        }
        if (partial.objectives) {
            this.objectives = partial.objectives;
        }
        if (partial.down) {
            this.down = partial.down;
        }
        if (partial.outOfAction) {
            this.outOfAction = partial.outOfAction;
        }
    }
}